lib/facade_converter.ts

```diff
@@ -0,0 +1,150 @@
+/// <reference path='../node_modules/typescript/bin/typescript.d.ts' />
+import base = require('./base');
+import ts = require('typescript');
+import ts2dart = require('./main');
+
+type FacadeHandler = (c: ts.CallExpression, context: ts.Expression) => void;
+
+export class FacadeConverter extends base.TranspilerStep {
```

Comment:
seems unintuitive that this is a base.TranspilerStep but it doesn't handle any nodes on its own, instead is just a helper. Is it really a subtype?